src/components/calculator-components/children.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable-line react/no-array-index-key */
```

Comment:
- [OPTIONAL] I suggest that you do not disable linters as they help us to write cleaner code